VideoLocker/src/main/java/org/edx/mobile/view/DiscussionAddPostFragment.java

```diff
@@ -161,6 +161,7 @@ public void onClick(View v) {
                 threadBody.setRawBody(body);
                 threadBody.setTopicId(((DiscussionTopicDepth) topicsSpinner.getSelectedItem()).getDiscussionTopic().getIdentifier());
                 threadBody.setType(discussionQuestion);
+                threadBody.setFollowing(true);  // MA-1380
```

Comment:
since we are sending a hardcoded value in this case, so, I thought it would be better to explain the reason for it :wink: